chrome/content/zotero/zoteroPane.js

```diff
@@ -4563,6 +4573,13 @@ var ZoteroPane = new function()
 	}
 	
 	
+	this.displayMissingImagesMessage = function () {
+		var ps = Components.classes["@mozilla.org/embedcomp/prompt-service;1"]
+								.getService(Components.interfaces.nsIPromptService);
+		ps.alert(null, "", Zotero.getString('pane.item.attachments.missingImages.text'));
```

Comment:
Since we're showing a prompt anyway, maybe it should just be `confirmEx()` and we should ask whether to continue or cancel.

Reply:
It means we need to tweak the prompt message. Currently it's "Notes with missing images cannot be copied.", though confirming still copies them. How about "Some note images are missing and cannot be copied. Do you want to continue?"

Reply:
Also, is there any specific reason to not just use `confirm()`?

Reply:
`confirmEx()` lets us have clearer button labels like "Copy Anyway". (I don't know how many different labels we'd need here for different situations. But even `confirmEx()` with `general.continue` would be better than the `OK` we get with `confirm()`.